kpi/views/environment.py

```diff
@@ -174,17 +175,26 @@ def process_other_configs(request):
         )
 
         # If the user isn't eligible for the free tier override, don't send free tier data to the frontend
-        if request.user.id and request.user.date_joined.date() > constance.config.FREE_TIER_CUTOFF_DATE:
-            data['free_tier_thresholds'] = {
-                'storage': None,
-                'data': None,
-                'transcription_minutes': None,
-                'translation_chars': None,
-            }
-            data['free_tier_display'] = {
-                'name': None,
-                'feature_list': [],
-            }
+        if request.user.id:
+            # default to checking the user's join date
+            date_joined = request.user.date_joined.date()
+            # if the user is in an organization, use the organization owner's join date instead
+            if request.user.organizations_organization:
```

Comment:
Could be simplified as one ORM query. I'm pretty sure you're doing a query on this very line. All you really want to fetch here is the date_joined (or null) of the org owner. Not 100% sure, but if you think about it from the org owner's perspective you might be able to simplify. Fetch that one row, filter away others, get the date. At a glance that sounds like one query and less logical statements.